CONTRIBUTING.md

```diff
@@ -90,6 +90,10 @@ jupyterlab_launcher` to get the latest version.
 
 * To install JupyterLab in isolation for a single conda/virtual environment, you can add the `--sys-prefix` flag to the extension activation above; this will tie the installation to the `sys.prefix` location of your environment, without writing anything in your user-wide settings area (which are visible to all your envs):
 
+* You can run `npm run build:main -- -- -- --devtool source-map` to build more accurate sourcemaps that show the original
```

Comment:
Are the extra `--` needed here?

Reply:
@blink1073 Yeah, because it calls `npm run` many times nested so it needs that many to get to the `webpack` command.

Reply:
@blink1073 created some more npm run command instead. How does it look now?